### ISSUE TITLE
Doesn't appear to fix LibreOffice on Windows 11 24H2

### ISSUE THREAD

**Issue by seagull** [WildByDesign/Aero.msstyles-win32-darkmode#3]
Happy to add whatever information you need, but in essence...

![image](https://github.com/user-attachments/assets/44e32a4c-3575-43b8-973f-6e06a1b9338c)

I followed the guide to the letter, even rebooted a second time.
Any help appreciated.

**Comment by WildByDesign** [comment]
Thanks for reporting this. I created a new version with the original 24H2 Aero msstyles file so that we have a new base for 24H2, replaced the one single button resource and updated it. I also updated ThemeTool to 3.0 which also included some important fixes for 24H2. It had stopped working on 24H2. So use the new ThemeTool included in this update.

Please try the following release:
https://github.com/WildByDesign/Aero.msstyles-win32-darkmode/releases/tag/24H2

I tested it myself and it is working well for me with LibreOffice. But if you run into any troubles, please let me know. There is a possibility that you might need to update the SecureUxTheme patch. It will likely say "Outdated" in ThemeTool near Installed. If you update it, you will want to switch to an original Windows (dark) theme first. You would have to run ThemeTool as Admin. You would click Uninstall to remove the SecureUxTheme patch, a reboot is needed, then run ThemeTool as Admin again when it boots up. This time choose Install (also, Remove DefaultColors must remain checked), it will require another reboot. At this point you're good to go. You can switch to the Aero11-24H2 (fixed aero) theme now.

**Comment by seagull** [comment]
![image](https://github.com/user-attachments/assets/76496309-797c-4a55-acb9-e12a3e7e6d0e)

It works! Thank you so much.

**Comment by WildByDesign** [comment]
Awesome, it looks great in your screenshot. I'm glad it worked.

**Comment by seagull** [comment]
Bad news - changes do not appear to survive a reboot. Even after re-applying the theme I get this now:

![image](https://github.com/user-attachments/assets/bebdf5b8-a14a-4053-b69a-9a140c04ee8b)

Any ideas?

**Comment by WildByDesign** [comment]
It seems like it’s reverting to the original Aero theme for some reason. I’m guessing SecureUxTheme is failing for some reason.

There is some good news though. LibreOffice devs just pushed a proper fix for the dark mode button issue and it looks great. It will be included in the very next stable update.

I will reopen this issue and take a look in the morning.

**Comment by WildByDesign** [comment]
Here's a couple of things to check for to ensure SecureUxTheme is working:

The following registry key and settings should exist:

```
[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\winlogon.exe]
"GlobalFlag"=dword:00000100
"VerifierDlls"="SecureUxTheme.dll"
```

The SecureUxTheme.dll should exist in the following location:

`C:\Windows\System32\SecureUxTheme.dll`

Also please check and make sure that `C:\Windows\System32\SecureUxTheme.dll` is not 0 bytes in size. When SecureUxTheme is uninstalled, it blanks that SecureUxTheme.dll file out. I just want to make that the SecureUxTheme.dll file exists and that it is not 0 bytes. It should be around 39KB for version 3.

**Comment by WildByDesign** [comment]
Another thing that we need to make sure is that the `SecureUxTheme.dll` module is loaded into the `winlogon.exe` process.

You can check with Process Explorer. It must run as Admin to be able to see into the `winlogon.exe` process.

Click on Find in the menu bar and then Find Handle or DLL. Search for `secureuxtheme` and ensure that it shows up loaded into `winlogon.exe`. It wont show if Process Explorer is not run as Admin.

**Comment by WildByDesign** [comment]
Anyway, the proper LibreOffice fix is merged and expected in the 24.8.3 release soon. There should be a release candidate (24.8.3 RC1) within the next week which I personally will use. Followed by final 24.8.3 release mid-November.

Once the fix is officially release, I will close this repo because it will no longer be necessary.

**Comment by WildByDesign** [comment]
@seagull LibreOffice_24.8.3.1_Win_x86-64.msi (release candidate) was just built and it contains the fix. I am using it right now and it looks great.

Link: https://dev-builds.libreoffice.org/pre-releases/win/x86_64/LibreOffice_24.8.3.1_Win_x86-64.msi

I hope it's okay with you but I am going to close this issue because I am much happier with an official, proper fix instead of my temporary fix which was just a workaround.
